### ISSUE TITLE
Icon: add article icon (for guides)

### ISSUE THREAD

**Issue by thebargaintenor** [pluralsight/design-system#740]
### Desired behavior

Add "article" icon currently only in the design system Figma.
![image](https://user-images.githubusercontent.com/6354333/70460357-16dfea80-1a84-11ea-9708-9f62b34ba71e.png)
SVG export from Figma:
[article.svg.zip](https://github.com/pluralsight/design-system/files/3941096/article.svg.zip)

### The Value Add

Search will be using it for Guide results

**Comment by jaketrent** [comment]
Hey @thebargaintenor , thanks for the request.

To help clarify whether this is an icon that belongs in the DS common set, 2 questions:

- Why not just use this icon from your Search apps src codebase?
- Do you know of other UIs now or in the future that include this icon?

**Comment by thebargaintenor** [comment]
@jaketrent I think the answer to the first is more about consistency with the other icons for content types, which were all represented in the DS already.

The explanation I got was that our designer discussed it with the designer for written content and agreed on that particular icon since it was available through the DS for Figma and neither was aware that icons for figma wouldn't already be available in the DS for dev teams.

tldr - Sounds like Written Content will be using it too?

**Comment by jaketrent** [comment]
Hi @thebargaintenor, thanks for your patience.  If I knew more, I wouldn't have to ask so many questions. :) . In your answer, I learned and then went to confirm that the `article` icon is in the common set in Figma but not yet in the code.  

We want to get all Figma common set into the Icon code common set so they're ready to use.  Our support of esm builds now enables us to do this with a clear conscience as it relates to bundle sizes.  

Yes, we'll get this done.

**Comment by jaketrent** [comment]
Related: https://github.com/pluralsight/design-system/issues/739
